src/EnergyPlus/StandardRatings.cc

```diff
@@ -362,12 +367,22 @@ namespace StandardRatings {
                 auto const SELECT_CASE_var(ChillerType);
 
                 if (SELECT_CASE_var == TypeOf_Chiller_ElectricEIR) {
+                    if (RedCapNum == 1.0) {
+                        // Get curve modifier values at rated conditions (load = 100%)
+                        ChillerCapFT_rated = CurveValue(state, CapFTempCurveIndex, EvapOutletTemp, CondenserInletTemp);
+                        ChillerEIRFT_rated = CurveValue(state, EIRFTempCurveIndex, EvapOutletTemp, CondenserInletTemp);
+
+                        // Report rated capacity and chiller COP
+                        PreDefTableEntry(state, state.dataOutRptPredefined->pdchMechRatCap, ChillerName, RefCap * ChillerCapFT_rated);
+                        PreDefTableEntry(state, state.dataOutRptPredefined->pdchMechRatEff, ChillerName, RefCOP / ChillerEIRFT_rated);
+                    }
+
                     // Get capacity curve info with respect to CW setpoint and entering condenser temps
                     ChillerCapFT = CurveValue(state, CapFTempCurveIndex, EvapOutletTemp, CondenserInletTemp);
 
                     ChillerEIRFT = CurveValue(state, EIRFTempCurveIndex, EvapOutletTemp, CondenserInletTemp);
 
-                    PartLoadRatio = ReducedPLR(RedCapNum) / ChillerCapFT;
+                    PartLoadRatio = ReducedPLR(RedCapNum) * ChillerCapFT_rated / ChillerCapFT;
```

Comment:
Take into account the chiller capacity modifier at rated conditions to correctly calculate a chiller's PLR based on the AHRI definition of load/percentage of load (as pointed out in #8192).

Reply:
I am puzzled by this code change here. The four distinct test *PartLoadRatio* are predefined per the AHRI test requirements.  I do not understand why you needed to adjust the **PartLoadRatio** values. In fact looking at the code lines where the *PartLoadRatio* is used (see the code snippet below), the capacity and power equations already account for capacity and power corrections needed. I think, we might be double counting for capacity correction. I do not agree with this change unless there is a justification. @lymereJ do you have justification why you need to adjust the part-load-ratio?

```
                AvailChillerCap = RefCap * ChillerCapFT;
                Power = (AvailChillerCap / RefCOP) * ChillerEIRFPLR * ChillerEIRFT;
                EIR = Power / (PartLoadRatio * AvailChillerCap);
```

Reply:
@Nigusse - Thanks for looking at this PR! The related issue was posted following an email conversation with @mjwitte. The justification is I believe provided in the issue but it might not be super clear and it doesn't include the example that we discussed. Here's the example:
- Let’s consider the “ElectricEIRChiller McQuay WSC 471kW/5.89COP/Vanes CAPFT” curve from the chiller dataset
- At AHRI rated conditions, the output of the curve is ~0.5
- The chiller part load ratio is calculated on line 370 of the function CalcChillerIPLV as:
(line 370)   PartLoadRatio = ReducedPLR(RedCapNum) / ChillerCapFT
where ReducedPLR(RedCapNum) appears to be the % loading specified in AHRI Std 550/590 (100%, 75%, 50% and 25%)
- Typically, 
(Eqn. 1)  PLR = Load / (Operating Capacity), and 
(Eqn. 2)  Operating Capacity = (Reference Capacity) * Cap-f-T_operating, 
- So line 370 implies 
(Eqn. 3)  Load = (AHRI % load) * (Reference Capacity) 
- Following the current approach, for the curve considered we would obtain a PLR = 2 for 100% loading
- However, the AHRI Std 550/590 defines the percent load(ing) as “The ratio of the part-load net capacity to the full-load rated net capacity at the full-load rating conditions”
- This seems to imply that:
(Eqn. 4)  Load = (AHRI % load) * (Rated Capacity), which is different than (Eqn. 3)
- If we consider the following:
(Eqn. 5)  Rated Capacity = (Reference Capacity) * Cap-fT_rated
(Eqn. 6)  Load = (AHRI % load) * (Reference Capacity) * Cap-f-T_rated
- Combining Eqns. 1, 2, and 6, we would expect line 384 to as follows:
(line 384, modified)   PartLoadRatio = ReducedPLR(RedCapNum) * ChillerCapFT_rated / ChillerCapFT_operating

Please, let me know if you still have concerned with the proposed changes. Thanks!

Reply:
See Section 3.11 for the definition of percent load: https://www.ahrinet.org/App_Content/ahri/files/STANDARDS/AHRI/AHRI_Standard_550-590_I-P_2015_with_Errata.pdf

Reply:
Looking at the definition of Percent Load from section 3.11 of AHRI Std 550/590, which is shown below:
*3.11 Percent Load (%Load). The ratio of the part-load net capacity to the full-load rated net capacity at the full-load rating
conditions, stated in decimal format. (e.g.100% = 1.0). The full load rating conditions, corresponding to 100% Load, shall be at Standard Rating Conditions. Exception: Configurable Units shall use Application Rating Conditions.*

I agree that the (AHRI % load) should percent load of the Full Load Capacity at the *Standard Rating Conditions*, which is  AHRI test conditions A as defined in Table 1 (Page 8 of the AHRI Std 550/590). There are also examples in the standard that demonstrate how the (AHRI % load) are calculated. @lymereJ I am convinced of the proposed change.